modules/org.restlet.ext.lucene/src/main/java/org/restlet/ext/lucene/SolrClientHelper.java

```diff
@@ -183,8 +184,7 @@ public void start() {
                     if (!config.exists()) {
                         config = new File(new URI(configFile));
                     }
-                    coreContainer = CoreContainer.createAndLoad(directory,
-                            config);
+                    coreContainer = CoreContainer.createAndLoad(Paths.get(directory), config.toPath());
```

Comment:
I'm pretty sure that we can get a path from a URI without using a File. 
Is `config` used elsewhere?

Reply:
sure, thanks Cyprien